pkg/ansible/watches/watches.go

```diff
@@ -297,13 +323,11 @@ func verifyGVK(gvk schema.GroupVersionKind) error {
 func verifyAnsiblePath(playbook string, role string) error {
 	switch {
 	case playbook != "":
-		playbookPath := getFullPath(playbook)
-		if _, err := os.Stat(playbookPath); err != nil {
+		if _, err := os.Stat(playbook); err != nil {
 			return fmt.Errorf("playbook: %v was not found", playbook)
 		}
 	case role != "":
-		rolePath := getFullRolePath(role)
-		if _, err := os.Stat(rolePath); err != nil {
+		if _, err := os.Stat(role); err != nil {
```

Comment:
Is it working when with `operator-sdk run --local` as well?

Reply:
Worked for me yes :)